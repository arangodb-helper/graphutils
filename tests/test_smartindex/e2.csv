_from|_to|_key
333|111|a
111|111|b
333|222|c
111|222|d
