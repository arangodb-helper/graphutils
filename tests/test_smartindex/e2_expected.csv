_from|_to|_key
v2/33:333|v1/11:111|33:a:11
v2/11:111|v1/11:111|11:b:11
v2/33:333|v1/22:222|33:c:22
v2/11:111|v1/22:222|11:d:22
